call get_netload() first on lo0 before get_netlist() otherwise the plugin blows with
[ERROR] glibtop_get_netload: Function not implemented

https://gitlab.xfce.org/panel-plugins/xfce4-systemload-plugin/-/issues/22
Index: panel-plugin/network.c
--- panel-plugin/network.c.orig
+++ panel-plugin/network.c
@@ -40,8 +40,10 @@ static gint
 read_netload_libgtop (gulong *bytes)
 {
     glibtop_netlist netlist;
+    glibtop_netload netload;
     char **interfaces, **i;
 
+    glibtop_get_netload (&netload, "lo0");
     interfaces = glibtop_get_netlist (&netlist);
     if (!interfaces)
         return -1;
@@ -49,7 +51,6 @@ read_netload_libgtop (gulong *bytes)
     *bytes = 0;
     for (i = interfaces; *i != NULL; i++)
     {
-        glibtop_netload netload;
         glibtop_get_netload (&netload, *i);
         *bytes += netload.bytes_total;
     }
